src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -93,18 +93,12 @@ function ReportActionItemMessageEdit(props) {
         return Str.htmlDecode(props.draftMessage);
     });
     const [selection, setSelection] = useState({start: 0, end: 0});
-    const [isFocused, setIsFocused] = useState(false);
     const [hasExceededMaxCommentLength, setHasExceededMaxCommentLength] = useState(false);
 
     const textInputRef = useRef(null);
     const isFocusedRef = useRef(false);
     const insertedEmojis = useRef([]);
 
-    useEffect(() => {
-        // required for keeping last state of isFocused variable
-        isFocusedRef.current = isFocused;
-    }, [isFocused]);
-
```

Comment:
Revert.

Reply:
Can you explain why this is needed? @s77rt

Reply:
The change is not needed. Let's just add the missing condition

Reply:
@s77rt 
reverted

Reply:
@s77rt 

Reverting raises an issue. In android chrome, text input gets focused after deletion and then draft is unmounted. So main composer gets hidden. When the draft is deleted, isFocusedRef.current is false even the input gets focused due to the effect delay.

Reply:
Using ref only, we can avoid this

Reply:
@s77rt How about using ref only solution? This works on all platforms